### ISSUE TITLE
Submit modifications for Node-HID upstream

### ISSUE THREAD

**Issue by ajfisher** [Makeblock-official/mbot_nodebots#3]
Submit the modifications for the wireless controller upstream so that default node-HID can be used as the package and not have a custom installer.

**Comment by lynnaloo** [comment]
Can you summarize what problems you had with the default `node-hid` module?

**Comment by ajfisher** [comment]
It didn't work out of the box - needed to be recompiled for use with this
module.
